lib/test.rb

```diff
@@ -0,0 +1,13 @@
+[1,2,3,4].each_with_index do |elem,index|
+puts "this is the #{index +1} iteration"
+puts "the element on which the block is beeing executed is #{elem}"
+puts "=======  "* 20 
+puts elem*elem
```

Comment:
Surrounding space missing for operator '*'.<br>Do not write to stdout. Use Rails' logger if you want to log.